test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua

```diff
@@ -0,0 +1,59 @@
+--  Requirement summary:
+--  [RegisterAppInterface] REJECT
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that App will be rejected with HMI type WEB_VIEW 
+--  when policy table is empty.
+--
+--  Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to SDL.
+--
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table is empty.
+--     SDL->appID: REJECT, success:"false":RegisterAppInterface()
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Test Configuration ]]
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = "WEB_VIEW"
+
+--[[ Local Functions ]]
+local function registerApp()
```

Comment:
@ydementieiev Please move this function into common module (it is used by more than one script) and rename it to better reflect its purpose.

Reply:
@aderiabin [moved](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/9271d45b0b4e176fad4cd4070eeb060facb11f43#diff-7b7ca1c3b1b899ba16ded7d51adceea0R119)